api/src/main/java/org/openmrs/module/appointmentscheduling/reporting/data/evaluator/AppointmentCreatorDataEvaluator.java

```diff
@@ -0,0 +1,14 @@
+package org.openmrs.module.appointmentscheduling.reporting.data.evaluator;
+
+import org.openmrs.annotation.Handler;
+import org.openmrs.module.appointmentscheduling.reporting.data.definition.AppointmentCreatorDataDefinition;
+
+@Handler(supports = AppointmentCreatorDataDefinition.class)
+public class AppointmentCreatorDataEvaluator extends AppointmentPropertyDataEvaluator {
+
+    @Override
+    public String getPropertyName() {
+        return "timeSlot.appointmentBlock.creator";
```

Comment:
Try "creator.person", I think that will work.  (also note that it will be getting the creator of the appointment, not the appointment block associated with the appointment, which is who we will want, I believe)

Reply:
Yes, this will likely work.  The way it is _supposed_ be done though, is that the Definition/Evaluator should return a User, not a String (i.e., just "creator" here), and then when this is added to a data set / report, it is added with an appropriate DataConverter instance that renders it the way you want.  In this case, a PropertyConverter on the "person" property, followed by an ObjectFormatter converter will render the full name of the person I believe.